lib/matplotlib/table.py

```diff
@@ -70,20 +70,21 @@ def __init__(self, xy, width, height, *,
             The cell width.
         height : float
             The cell height.
-        edgecolor : color
+        edgecolor : color, default: 'k'
             The color of the cell border.
-        facecolor : color
+        facecolor : color, default: 'w'
             The cell facecolor.
-        fill : bool
+        fill : bool, default: True
             Whether the cell background is filled.
-        text : str
+        text : str, optional
             The cell text.
-        loc : {'left', 'center', 'right'}, default: 'right'
+        loc : {'right', 'center', 'left'}
             The alignment of the text within the cell.
-        fontproperties : dict
+        fontproperties : dict, optional
             A dict defining the font properties of the text. Supported keys and
             values are the keyword arguments accepted by `.FontProperties`.
-        visible_edges : str, default: 'closed'
+        visible_edges : {'closed', 'open', 'horizontal', 'vertical'} or \
+            substring of 'BRTL'
```

Comment:
Should not be indented if wrapped:
```suggestion
        visible_edges : {'closed', 'open', 'horizontal', 'vertical'} or \
substring of 'BRTL'
```

Reply:
fixed